Bottom: 5b93c85fbebff45326b607a0757ca424b88300a0
Top:    c0cf73cdfaee9d00ee452bd869307fdbb9325ae4
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-12 14:44:42 +0800

Refresh of vhost-optmize-vhost_signal

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 5d2f627..d3fa550 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2188,8 +2188,7 @@ static bool vhost_notify(struct vhost_dev *dev, struct vhost_virtqueue *vq)
 	/* We're sure if the following conditions are met, there's no
 	 * need to notify guest:
 	 * 1) cached used event is ahead of new
-	 * 2) old to new updating does not cross cached used event.
-	 */
+	 * 2) old to new updating does not cross cached used event. */
 	if (vring_need_event(vq->last_used_event, new + vq->num, new) &&
 	    !vring_need_event(vq->last_used_event, new, old))
 		return false;
